<launch>
				<param name="velocity_depth_default" value="0.06" />

    <node name="state_feedback" pkg="seabot_depth_regulation" type="state_feedback_node" output="screen" ns="regulation" respawn="true" respawn_delay="5">
								<param name="frequency" value="5.0" />

								<param name="root_regulation" value="-1.0" />

								<param name="delta_velocity_lb" value="-0.01" />
								<param name="delta_velocity_ub" value="0.01" />

        <param name="hysteresis_piston" value="0.6" />
								<param name="limit_depth_regulation" value="0.3" />
								<param name="speed_volume_sink" value="4.0" />
<!--								<param name="speed_volume_sink" value="4.0" />-->
    </node>

    <node name="waypoint_node" pkg="seabot_waypoint_regulation" type="waypoint_node" output="screen" ns="regulation" respawn="true" respawn_delay="5">
        <param name="frequency" value="10.0" />
        <param name="delta_valid_time" value="3.0" />
        <param name="coeff_P" value="1.0" />
        <param name="hysteresis_circle_in" value="10.0" />
        <param name="hysteresis_circle_out" value="30.0" />
        <param name="linear_speed" value="1.0" />
        <param name="depth_limit_switch_off" value="0.5" />
    </node>

</launch>

